opdracht4.js

```diff
@@ -0,0 +1,103 @@
+// Opdracht 4a: Zorg ervoor dat er een string wordt gegenereerd voor de naam van een tv. Maak een functie die één enkel tv-object (zoals inventory[0] of inventory[6])
+// verwacht en de naam op de volgende manier samenvoegt: [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV.
+// Test of jouw functie ook werkt wanneer er een ander tv object wordt meegegeven.
+
+function tvBrands(tv) {
+    let branded = "";
+    for (let i = 0; i < tv.length ; i++) {
+        branded = `${inventory[i].brand}  ${inventory[i].type} - ${inventory[i].name}`
+    }
```

Comment:
`tv` is een string. Nu itereer je over elk karakter van die string. Verder overschrijf je telkens de string opnieuw, dus is het sowieso niet nodig om over die `tv`-string te itereren.